src/main/resources/static/api.yaml

```diff
@@ -1949,6 +2116,43 @@ components:
         keyValue:
           type: string
           format: string
+    VerifyOtpRequest:
+      type: object
+      required:
+        - sessionId
+        - value
+      properties:
+        sessionId:
+          type: string
+          format: uuid
+          example: 3fa85f64-5717-4562-b3fc-2c963f66afa6
+        value:
+          type: string
+          example: 666666
+          description: '6 digit number'
+    VerifyOtpResponse:
+      type: object
+      properties:
+        temporaryToken:
+          type: string
+          example: eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJhMjQ2NWRmYy1iY2Q2LTRmNTItYTQxYS03MjA4Mjg3YmE1NWMiLCJleHAiOjE1ODgxODIzMDIsImlhdCI6MTU4ODE2NDMwMn0.CmOpT1Q-s05WY9aq5YbAz-SX9AgaypzrxA273mxyyUSvOqtKbZCIruBLoDg61PK2tCY_NbHY-8mWkshYFP-IYQ
+          description: JWT Token
+    GenerateOtpRequest:
+      type: object
+      required:
+        - username
+      properties:
+        username:
+          type: string
+          format: string
+          example: raj.sharma@ncg
+    GenerateOtpResponse:
+      type: object
+      properties:
+        sessionId:
+          type: string
+          format: uuid
+          example: 3fa85f64-5717-4562-b3fc-2c963f66afa6
```

Comment:
We need to return some meta info as well, on which medium (email/phone), on what number/email (masked), if possible (expiry) too. that can be shown on the UI;

Reply:
1. How would consent-manager be aware of otp expiry time that is configured on otp-service? POST "/otp" of otp-service is not returning anything in the response. 

2. Should we make this expiry time as configurable value in consent-manager?

Reply:
I have added these fields also. Made `expiryInMinutes` as a configuration under consentmanager.otpservice. Added it as a different commit.